draft-ietf-cats-metric-definition.md

```diff
@@ -95,7 +102,17 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 # Definition of Metrics
 
-Introducing a definition of metrics requires balancing the following trade-off: if the metrics are too fine-grained, they become unscalable due to the excessive number of metrics that must be communicated through the metrics distribution protocol. (See {{?I-D.rcr-opsawg-operational-compute-metrics}} for a discussion of metrics distribution protocols.) Conversely, if the metrics are too coarse-grained, they may not have sufficient information to enable proper operational decisions. To ensure scalability while providing sufficient detail for effective decision-making, this document provides a definition of metrics that incorporates three levels of abstraction:
+## Problem Statement - Why Three Metric Levels?
```

Comment:
Instead of "Problem Statement - Why Three Metric Levels?"

suggest the following heading:

"Design Principles: Why Three Metric Levels?"